### ISSUE TITLE
Custom rule elements on ammunition are ignored

### ISSUE THREAD

**Issue by johncarney** [JDCalvert/FVTT-PF2e-Ranged-Combat#183]
Custom rule elements on ammunition are ignored when the _PF2e Ranged Combat_ module is active.

# Steps to reproduce

1. Create and launch a PF2e test world. 
2. Ensure that the _PF2e Ranged Combat_ module is NOT active in the test world.
3. Create an actor in the test world and equip them with a longbow and some arrows.  
4. Draw the longbow.
5. Load the arrows onto the longbow. 
6. Add the following `AdjustStrike` rule element to the arrows.
```
{
    "key":        "AdjustStrike",
    "mode":       "add",
    "property":   "materials",
    "value":      "cold-iron",
    "definition": ["item:id:{item|id}"]
}
```
7. Now in the action tab on the actor's sheet, click on the longbow's `DAMAGE` button. You should see that the damage has the `COLD IRON` trait as in the screenshot below.
<img width="288" alt="image" src="https://github.com/JDCalvert/FVTT-PF2e-Ranged-Combat/assets/322950/833ca872-18e4-4c03-a540-1e84e87bd5ac">

8. Now enable the _PF2e Ranged Combat_ module and roll the longbow's `DAMAGE` button again. The `COLD IRON` trait no longer appears on the damage card.
<img width="288" alt="image" src="https://github.com/JDCalvert/FVTT-PF2e-Ranged-Combat/assets/322950/1658dcbb-52a5-4851-a2a9-99b947267be5">

**Comment by TheGnome1** [comment]
I tested this and see the same issue

**Comment by JDCalvert** [comment]
This should now be fixed in 6.2.1.
